lib/Doctrine/ODM/PHPCR/Mapping/ClassMetadata.php

```diff
@@ -1561,6 +1562,24 @@ public function getFieldValue($document, $field)
     }
 
     /**
+     * Gets the mapping of a (regular) field that holds some data but not a
+     * reference to another object.
+     *
+     * @param string $fieldName The field name.
+     *
+     * @return array The field mapping.
+     *
+     * @throws MappingException
+     */
+    public function getFieldMapping($fieldName)
+    {
+        if ( ! isset($this->fieldMappings[$fieldName])) {
+            throw MappingException::mappingNotFound($this->name, $fieldName);
+        }
+        return $this->fieldMappings[$fieldName];
```

Comment:
Would this work? From what I see, `fieldMappings` is an array of field names,e.g. `array('fieldOne', 'fieldTwo')` not an associative array of field mappings.